features/webdriver-sensors.yml.dist

```diff
@@ -0,0 +1,14 @@
+# Generated from: webdriver-sensors.yml
```

Comment:
This file should no longer exist. `dist` should've warned you about this but didn't, so I opened https://github.com/web-platform-dx/web-features/pull/1681.

Reply:
bah, i `mv`'d the feature file but then manually added the dist file, thanks for catching!